server/cache.js

```diff
@@ -17,6 +18,11 @@ module.exports = Cache;
 
 function Cache(ecs){
   Emitter.call(this);
+  this.tasks = LRU({
+    max: 1000000,
+    maxAge: ms('1d'),
```

Comment:
Ahhh, I think you're right! Sorry, it's been awhile since I've touched this codebase.

A day sounds good to me!